sszgen/hash.go

```diff
@@ -181,7 +215,7 @@ func (v *Value) hashTreeRootContainer(start bool) string {
 
 	out := []string{}
 	for indx, i := range v.o {
-		str := fmt.Sprintf("// Field (%d) '%s'\n%s\n", indx, i.name, i.hashTreeRoot())
+		str := fmt.Sprintf("// Field (%d) '%s'\n%s\n", indx, i.name, i.hashTreeRoot("", false))
```

Comment:
Adding a quick in-line comment to explain the "", false could help the reader, something like
```
i.hashTreeRoot("" /* ... */, false /* ... */)
```

Reply:
updated, ptal